core/templates/dev/head/components/rubrics-editor/rubrics-editor.directive.ts

```diff
@@ -47,14 +48,17 @@ angular.module('oppia').directive('rubricsEditor', [
         '/components/rubrics-editor/rubrics-editor.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$scope', '$filter', '$uibModal', '$rootScope',
-        'RubricObjectFactory', 'EVENT_SKILL_REINITIALIZED',
+        '$scope', '$filter', '$uibModal', '$rootScope', 'ContextService',
+        'RubricObjectFactory', 'EVENT_SKILL_REINITIALIZED', 'PAGE_CONTEXT',
         function(
-            $scope, $filter, $uibModal, $rootScope,
-            RubricObjectFactory, EVENT_SKILL_REINITIALIZED) {
+            $scope, $filter, $uibModal, $rootScope, ContextService,
+            RubricObjectFactory, EVENT_SKILL_REINITIALIZED, PAGE_CONTEXT) {
           var ctrl = this;
           ctrl.activeRubricIndex = 0;
           ctrl.explanationEditorIsOpen = [false, false, false];
+          ctrl.newSkillBeingCreated = (
```

Comment:
Do you have any other way to figure out this value -- perhaps by passing it as an attr into the directive? I think using ContextService is too indirect and it's possible that the rubric editor may be shown in other pages in the future.

Reply:
Yeah, passing it in as an attribute is possible, will do that.

Reply:
Done